de/reservation-form.html

```diff
@@ -408,7 +408,7 @@ <h3>Zimmertyp</h3>
 
 
           <div class="ef-formgroup form-group">
-            <h3>Kommentare</h3>
+            <h3><label for="comments">Kommentare</label></h3>
```

Comment:
This addition doesn't seem to have the desired effect (Firefox 69.0.1); besides, it appears to be missing in the english translation.

Reply:
I'd appreciate if you could elaborate. I could not notice a difference besides better accessibility.

Reply:
Maybe it's a result of the dependencies not being linked anywhere. I have no idea if the files I found are correct, as the site does look a bit scuffed. Would be nice to have a better explanation than just the .gitignore.